4/john-wiegley-captions.srt

```diff
@@ -1827,7 +1834,7 @@ I need to pack the string.
 
 418
 00:34:19,500 --> 00:34:29,700
-inaudible
+[inaudible]

```

Comment:
I hear:

    OC: I guess parse is a pure function
    JW: Yeah, we're working with pure functions here